.gitignore

```diff
@@ -15,3 +15,4 @@ bin/
 
 /text-ui-test/ACTUAL.TXT
 text-ui-test/EXPECTED-UNIX.TXT
+/data/
```

Comment:
Good choice to include local data storage path in .gitignore to prevent unnecessary files from being committed to the repository